src/main/java/org/gridsuite/study/server/dto/dynamicsimulation/solver/SolverInfos.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.study.server.dto.dynamicsimulation.solver;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.UncheckedIOException;
+
+import java.util.List;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@JsonTypeInfo(
+        use = JsonTypeInfo.Id.NAME,
+        property = "type",
+        include = JsonTypeInfo.As.EXISTING_PROPERTY,
+        visible = true
+)@JsonSubTypes({
+    @JsonSubTypes.Type(value = IdaSolverInfos.class, name = "IDA"),
+    @JsonSubTypes.Type(value = SimSolverInfos.class, name = "SIM")})
+public interface SolverInfos {
+    String getId();
+
+    SolverTypeInfos getType();
+
+    static List<SolverInfos> parseJson(String json) {
```

Comment:
this is not like our other json serialization/deserialization, we usually register the modules in springboot (using @configuration in main class)

Reply:
To discuss later..

At this moment, this Dto conformes to the standard parsed by default ObjectMapper of jackson.. In my opinion, it is not necessary to write a  prop serialization/deserialization.